modules/cnf-verify-sriov-operator-n3000.adoc

```diff
@@ -0,0 +1,232 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="verify-the-fpga-operator_{context}"]
+= Verifying application POD Access and Usage of FPGA on OpenNESS Network Edge
+
+To verify all the features are working together (SRIOV binding - device plugin - BBDEV config) and the functionality of the SR-IOV (FEC) VF inside a non-root pod, build a Docker image and run a simple validation application for the device.
+
+The automation of the Docker image build is available from the Edge Apps package. The image must be built on the same node that it is meant to be deployed or a server with the same configuration as the node that will run the workload. This is due to the Kernel dependencies of DPDK during the application build.
+
+An example of a prepared sample application's docker image can be found in Intel® OpenNESS' project github EdgeApps repo. OpenNESS is an edge computing software toolkit that enables highly optimized and performant edge platforms to on-board and manage applications and network functions with cloud-like agility across any type of network.
+For more information, go to link:http://www.openness.org[openess.org].
+
+.Prerequisites
+
+* Intel® FPGA PAC N3000 card (Optional)
+* Node/s installed with n3000-operator
+* Node/s installed with SRIOV-FEC operator
+* RT Kernel configured with Performance Addon Operator
+
+.Procedure
+
+[NOTE]
+====
+All the commands run in the `vran-acceleration-operators` namespace.
+====
+
+. Populate a YAML file for example `pod-test.yaml` with the following:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-bbdev-sample-app
+spec:
+  containers:
+  - securityContext:
+      privileged: false
+      capabilities:
+        add:
+          - IPC_LOCK
+          - SYS_NICE
+    name: bbdev-sample-app
+    image: bbdev-sample-app:1.0
+    command: [ "sudo", "/bin/bash", "-c", "--" ]
+    args: [ "while true; do sleep 300000; done;" ]
+    volumeMounts:
+    - mountPath: /hugepages
+      name: hugepage
+    - name: class
+      mountPath: /sys/devices
+      readOnly: false
+    resources:
+      requests:
+        hugepages-1Gi: 4Gi
```

Comment:
lets add a <1> explaining that hugepages needs to be configure via PAO and also isolated CPUs